splink/linker.py

```diff
@@ -2104,41 +2108,25 @@ def cluster_pairwise_predictions_at_threshold(
 
         return cc
 
-    def _compute_cluster_metrics(
+    def _compute_metrics_nodes(
         self,
         df_predict: SplinkDataFrame,
         df_clustered: SplinkDataFrame,
         threshold_match_probability: float,
-    ):
-        """Generates a table containing cluster metrics and returns a Splink dataframe
-
-        Args:
-            df_predict (SplinkDataFrame): The results of `linker.predict()`
-            df_clustered (SplinkDataFrame): The outputs of
-                `linker.cluster_pairwise_predictions_at_threshold()`
-            threshold_match_probability (float): Filter the pairwise match predictions
-                to include only pairwise comparisons with a match_probability above this
-                threshold.
-
-        Returns:
-            SplinkDataFrame: A SplinkDataFrame containing cluster IDs and selected
-            cluster metrics
-
-        """
-
-        # Get unique id columns
+    ) -> SplinkDataFrame:
```

Comment:
I think it could be useful to have a simple docstring here describing the shape of the data, e.g. one row per node etc, just to make it easy for people to imagine what's being produced

Reply:
have added a docstring here and for `_compute_metrics_clusters` - let me know if you think that has the info that is useful

Reply:
Thanks! Both look great and the tables you've included really help :)